vpplink/generated/vpp_clone_current.sh

```diff
@@ -92,12 +92,9 @@ function git_clone_cd_and_reset ()
 
 # --------------- Things to cherry pick ---------------
 
-git_clone_cd_and_reset "$1" 7419bede7ad73544338fd4363da833b2d5fc89a5 # misc: Initial 24.02-rc0 commit (tag: v24.02-rc0)
+git_clone_cd_and_reset "$1" 2ae8f79b5087c8da7d30283693f22ca710a60347 # misc: Initial 24.02-rc0 commit (tag: v24.02-rc0)
```

Comment:
Can you also update the comment ( `# misc: Initial 24.02-rc0 commit (tag: v24.02-rc0)` ) with the new VPP commit message header ?